fix: n-loading-bar use error and export some type (#511)
TuSimple/naive-ui: CHANGELOG.en-US.md

- `n-slider` add `format-tooltip` prop.
- `n-upload` add `event` in `on-finish` callback params.
- `n-rate` add `readonly` prop.
- `n-notification` export `NotificationApi`, `NotificationOptions` and `NotificationReactive` type.

### Fixes

- Fix `n-pagination` page count is not correct when `item-count` is 0.
- Fix `n-scrollbar` `content-style` can not override the default width of style.
- Fix `n-select` placeholder transition.

> - Fix `n-loading-bar`　use `useLoadingBar` the result is undefined.

Reviewer: ```suggestion
- Fix `n-loading-bar` `useLoadingBar`'s return type can be undefined.
```
